### PR TITLE
Add tryharder

### PR DESCRIPTION
**Pull request by RealJammy**
beanman I'm not looking for an ez pr, pwease merge uwu

### CODE REVIEW

**Comment by RealJammy** [comment]
fuck i noticed a spelling mistake on Line 591, oopsies

File: cdbot/cogs/fun.py [hunk 1]
```diff
@@ -584,6 +584,13 @@ async def flowchart(self, ctx: Context):
         Sends the image of the challenge solving flowchart.
         """
         await ctx.send("https://cdn.discordapp.com/attachments/411573884597436416/767122366521278474/trythis.png")
+    
+    @command()
+    async def tryharder(self, ctx: Context):
+        """
+        Sends the tryharder video, make by OFFSEC.
```
Comment (RealJammy) [inline cdbot/cogs/fun.py:591]: fuck i spelt this wrong

**Comment by H4ckerJ4cker** [comment]
Duplicate of #560

**Comment by RealJammy** [comment]
😔 my code is backed by the jam working code guarantee tho jacker ⭐

**Comment by H4ckerJ4cker** [comment]
Can't argue against that.

**Review by Sh3llcod3 — APPROVED** [review]
I don't know if @thebeanogamer wants this but sure.

**Comment by thebeanogamer** [comment]
Duplicate of #560

**Pull request closed without merging**